### ISSUE TITLE
Scene Understanding demo is dependent on Spatial Awareness demo

### ISSUE THREAD

**Issue by MaxWang-MS** [microsoft/MixedRealityToolkit-Unity#9431]
## Describe the bug

In MRTK 2.6, a script in the Scene Understanding demo is referencing another script in the Spatial Awareness demo, which means when MRTK is consumed via UPM/Mixed Reality Feature Tool the Spatial Awareness sample in the examples package must be imported before importing the experimental Scene Understanding sample or otherwise there will be compiler errors in the console. This issue does not affect people using MRTK via unitypackages.

**Comment by ernijsyoyo** [comment]
Hi Max,

Thanks for information although the instructions are not fully clear to me.

1. I'm using the MR Feature Tool for and Unity Package Manager for integrating MRTK in my project
2. I've imported MRTK/Examples *Demos - SpatialAwareness* demo as you've mentioned
3. Scene Understanding does not show up as an example scene that I can import and I am getting compiler errors from Unity 

Also, I tried integrating MRTK 2.6.0 with the raw Unity packages and there I could not get the demo to work -> I.e. when running the example scene, the pre-serialized scene was not being rendered and Unity threw me an error saying that the Scene Understanding has not bee properly installed and I should use the MR Feature tool.

Screenshots below of my package manager window and error message from Unity

![image](https://user-images.githubusercontent.com/40242518/109957882-d6523980-7ce5-11eb-9821-4aa599bf7b2a.png)

![image](https://user-images.githubusercontent.com/40242518/109957947-ecf89080-7ce5-11eb-8314-e00f83cca59b.png)
